packages/core/src/components/drawer/drawer.tsx

```diff
@@ -210,6 +179,8 @@ export class Drawer {
         ref={(el) => (this.divElement = el as HTMLElement)}
         data-testid="container"
         id="div-container"
+        role="dialog"
+        aria-modal="true"
```

Comment:
```
Modal dialogs are when content is displayed and the user's interaction is limited to only that section until it is dismissed.
```
Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-modal

`aria-modal` is here not correct, even if the drawer is visible to elements in the background are still accessible